platform/linux-generic/pktio/dpdk.c

```diff
@@ -542,7 +542,8 @@ static inline int mbuf_to_pkt(pktio_entry_t *pktio_entry,
 		if (pktio_cls_enabled(pktio_entry))
 			copy_packet_cls_metadata(&parsed_hdr, pkt_hdr);
 		else if (parse_layer != ODP_PROTO_LAYER_NONE)
-			packet_parse_layer(pkt_hdr, parse_layer);
+			packet_parse_layer(pkt_hdr, parse_layer,
+					   (odp_proto_chksums_t)0U);
```

Comment:
Useful to have this here and elsewhere as a `#define`?

Reply:
Agree. XXX_NO_CHKSUMS is better than explicit casts.

Reply:
A define should be added for "(odp_proto_chksums_t)0U"  - e.g. PARSE_NO_CHKSUM.

Reply:
@psavol rebased PR, so that NO_CHKSUM is unnecessary now.